### PR TITLE
UCA 15.1d3 may02

### PR DESCRIPTION
**Pull request by markusicu**
From Ken:

... d3 of my DUCET generation.
These now include the addition of the 5 new IDC's, the move of 10A7F,
and the synthetic decomposition of 20A7 PESETA SIGN.

I haven't yet done the quotation mark work.

The CJK Extension I addition is basically a no-op for DUCET itself, but
does impact the sifter code a little, as well as the footer text for the
14651 CTT table generation.

### CODE REVIEW

File: unicodetools/src/main/java/org/unicode/text/UCA/WriteCollationData.java [hunk 1]
```diff
@@ -2133,6 +2133,10 @@ private static UCA buildCldrCollator(boolean addFFFx) {
             final int primary = ceCount < 1 ? 0 : CEList.getPrimary(output[0]);
             final int cat = Default.ucd().getCategory(i);
 
+            if (i == 0x20A7) {
+                // Starting with Unicode 15.1, U+20A7 PESETA SIGN sorts like "Pts".
```
Comment (eggrobin) [inline unicodetools/src/main/java/org/unicode/text/UCA/WriteCollationData.java:2137]: Note that UTC has not yet decided that ; we have made progress on discussing https://github.com/unicode-org/properties/issues/99, but we have not yet recommended anything.

Comment (Ken-Whistler) [reply unicodetools/src/main/java/org/unicode/text/UCA/WriteCollationData.java:2137]: @eggrobin Noted, but I am trying to lean forward in DUCET on what seemed to be the consensus from that discussion. It is easy enough to pull it back if people dump on the change during beta review.

**Review by Ken-Whistler — APPROVED** [review]
Changes look correct to me.

**Comment by markusicu** [comment]
FYI:
Before this PR, I should have finished & merged https://github.com/unicode-org/unicodetools/pull/403 but I forgot :-(
See there for the d1 and d2 changes.

**Pull request merged**